docs/developer/channels.mdx

```diff
@@ -253,6 +271,104 @@ Response:
 }
 ```
 
+## Reorder warehouses within channels
+
+The warehouses assigned to the channel can be sorted. The provided order defines the
+warehouses' order used in `PRIORITIZE_SORTING_ORDER` allocation strategy.
```

Comment:
`allocation strategy` could be linked to https://saleor-docs-mguwbmz3t-saleorcommerce.vercel.app/docs/3.x/developer/stock-allocation#allocation-strategies

Reply:
But again, in the end, there is a sentence with linked to the allocation section, so it won't be too much if I will repeat it here?

Reply:
But this chapter is about channel settings, so you might be not aware of the concept in that is in a different chapter